docs/docsite/rst/getting_started_ee/yaml/execution-environment.yml

```
@@ -2,9 +2,11 @@ version: 3
 
 images:
   base_image:
-    name: quay.io/fedora/fedora:39
+    name: registry.fedoraproject.org/fedora:42
 
 dependencies:
+  python_interpreter:
+    package_system: "python3"
```

I don't think this needs to be quoted.

```suggestion
    package_system: python3
```

---

Tested and confirmed quotes unneeded. Fix inbound